odps/config.py

```diff
@@ -83,7 +83,12 @@ def __init__(self, *args, **kw):
     def getvalue(self, silent=False):
         if self._use_pd:
             import pandas as pd
-            from pandas.core.config import OptionError as PandasOptionError
+            if pd.__version__ < '0.25.0':
```

Comment:
We could try import `pandas.core.config`, if failed, try to import `pandas._config.config`. Documents can be added about pandas version.

Reply:
Yes, we can do that now.

But it seems not right to handle compatibility issues with exceptions (exceptions are most likely to be something go unexpected).  

`OptionError` is private in '0.25.0' , but may be published and move to another place afterwards. We can't handle updates directly with `ImportError` then. 

Any idea?

Reply:
Could you given some more reasoning? I really can't understand why try except is preferred in this use case. Importing from  `pandas.core.config` will sure fail in current and future versions.

And code already committed here did fix the issue. 

See also:

https://github.com/pandas-dev/pandas/issues/27553
https://github.com/pandas-dev/pandas/issues/27656